### DISCUSSION TITLE
Zed & phpactor

### DISCUSSION THREAD

**Discussion by gldtn** [phpactor/phpactor#2421] [Q&A]
Has anyone tried to use phpactor with the new Zed editor yet?

https://zed.dev/
https://github.com/zed-industries

**Comment by dantleech** [comment]
this one? https://github.com/zededitor/zed 

not that i'm aware.

**Comment by gldtn** [comment]
> this one? https://github.com/zededitor/zed
> 
> not that i'm aware.

I'm sorry I forgot to link to it, updated  my first post to include the link. It's still in beta and they have not open sourced their code just yet, but according to the team it should be coming soon. 

https://zed.dev/
